assets/scss/main.scss

```diff
@@ -60,3 +60,28 @@ footer {
         max-width: 100%;
     }
 }
+
+table {
+    border: 1px solid lightgray;
+    margin: 1em;
+    padding: .5em;
+    tr {
+        border: 1px solid lightgray;
+        margin-left: .5em;
+        margin-right: .5em;
+        padding: .5em;
+    }
+    th {
+        border: 1px solid #495057;
+        margin-left: .5em;
+        margin-right: .5em;
+        padding: .5em;
+        background: lightgray;
+    }
+    td {
+        border: 1px solid lightgray;
+        margin-left: .5em;
+        margin-right: .5em;
+        padding: 5px;
+    }
+}
```

Comment:
This is updating styling for _all_ tables. I'm assuming that is intended?

Reply:
In particular, compare: https://deploy-preview-288--knative.netlify.app/docs/eventing/sources/#knative-sources vs https://knative.dev/docs/eventing/sources/#knative-sources

Can we revert this one file?

Reply:
Sorry, just got out of a block of meetings. Yes. Will revert now.

Thanks for pointing this out. It looks like the table CSS was not working correctly for me locally. Initially, tables were rendering blank for me, thus adding borders. But sadly I dropped the ball here and failed to circled back around to testing this in the Netlify Preview builds.